package.json

```diff
@@ -1974,6 +1974,11 @@
                 "title": "%AWS.command.auth.addConnection%",
                 "category": "%AWS.title%"
             },
+            {
+                "command": "aws.auth.showConnectionsPage",
+                "title": "%AWS.command.auth.showConnectionsPage%",
+                "category": "%AWS.title%"
+            },
```

Comment:
You shouldn't need to add anything to `package.json` until we start changing the UI. Add a command to `src/dev/activation.ts` or create an experimental flag (see the `Experiments` class).

We don't want to surface anything unfinished to users. If this makes things hard to work with then we can use a feature branch.